test/Polly.Caching.Tests/HybridCacheResiliencePipelineBuilderExtensionsTests.cs

```diff
@@ -15,7 +15,11 @@ public async Task NonGeneric_AddHybridCache_BuildsAndCaches()
         using var provider = services.Services.BuildServiceProvider();
         var cache = provider.GetRequiredService<HybridCache>();
 
-        var options = new HybridCacheStrategyOptions { Cache = cache, CacheKeyGenerator = _ => "builder-key" };
+        var options = new HybridCacheStrategyOptions
```

Comment:
I only noted on the one occurrence, but could you fix them all in the code you've added please?